src/clj/athens/self_hosted/web/presence.clj

```diff
@@ -37,6 +38,10 @@
       (clients/send! channel
                      (common-events/build-presence-all-online-event max-tx
                                                                     (clients/get-clients-usernames)))
+      (doseq [{:keys [username block-uid]} (vals @all-presence)]
+        (let [broadcast-presence-editing-event
+              (common-events/build-presence-broadcast-editing-event max-tx username block-uid)]
+          (clients/broadcast! broadcast-presence-editing-event)))
```

Comment:
This is too loud.
Did you maybe mean `clients/send!`?

Maybe you just wanted to send it to just connected client?
This happens already above.

Reply:
Yes, I did mean that, changed now.

The connected client is getting the list of usernames above, but is not getting the current list of block presences.